### ISSUE TITLE
Test failure with OpenBLAS 0.3.13

### ISSUE THREAD

**Issue by rmcgibbo** [phonopy/phonopy#140]
Hi,

I noticed that the tests for `phonopy` that are run during packaging for [NixOS](https://nixos.org/) recently started failing.

Here's a [log](https://nix-cache.s3.amazonaws.com/log/gw3ps8vrjrbi7qrlw5fgz585sdxn4jan-python3.7-phonopy-2.8.1.drv) from the failure. The failure is in `test_random_displacements`. It's completely deterministic, and I was able to git bisect the error down to the upgrade of openblas, our BLAS provider for numpy, from 0.3.12 to 0.3.13. That change alone takes the test from passing to failing.

It's completely possible that this is a bug in openblas or something, but I wanted to give you a heads up. I haven't looked into it further than that. Perhaps the test is numerically "borderline", and a switch in openblas numerics was sufficient to switch it over?

**Comment by atztogo** [comment]
Thank you very much for your report. I also struggled with this test and agree with you that the result can depend on the eigensolver (finally BLAS). I thought that the problem comes from degenerated solutions that give random combinations of eigenvectors, which results in different atomic displacements. So I chose non-degenerate crystal structure as the test system, but the problem still happened. Probably even for non-degenerate solutions, some eigenvectors are numerically correlated and changed by a tiny difference of the eigensolver. So I gave up using any eigenvsolver in this test. The latest test in v2.9.1 uses the pre-determined eigensolutions. So I hope it will not fail in your system, too.
